WordPressUI/Extensions/UIImageView+Gravatar.swift

```diff
@@ -110,15 +136,29 @@ extension UIImageView {
     /// P.s.:
     /// Hope buddah, and the code reviewer, can forgive me for this hack.
     ///
-    @objc
-    public func overrideGravatarImageCache(_ image: UIImage, rating: GravatarRatings, email: String) {
-        guard let gravatarURL = gravatarUrl(for: email, size: gravatarDefaultSize(), rating: rating.stringValue()) else {
+    @available(*, deprecated)
+    @objc public func overrideGravatarImageCache(_ image: UIImage, rating: GravatarRatings, email: String) {
+        guard let gravatarURL = gravatarUrl(for: email, size: gravatarDefaultSize(), rating: rating) else {
             return
         }
 
+        listenForGravatarChanges()
         overrideImageCache(for: gravatarURL, with: image)
     }
 
+    /// Updates the gravatar image for the given email, and notifies all gravatar image views
+    ///
+    /// - Parameters:
+    ///   - image: the new UIImage
+    ///   - email: associated email of the new gravatar
+    @objc public func updateGravatar(image: UIImage, email: String?) {
```

Comment:
I'm gonna jump in with a question. I'm curious as to why the `email` is optional. Two lines down the `guard` checks for it, so it _appears_ to be necessary.

Reply:
I flip-flopped on this: when called in `EpilogueUserInfoCell` the email is optional. It either needs a guard here or there. I thought putting it here makes `@objc` interaction easier, but I could be convinced to move it.